website/docs/r/data_export_rule.html.markdown

```diff
@@ -0,0 +1,41 @@
+---
+subcategory: "Data Export Rules"
+layout: "lacework"
+page_title: "Lacework: lacework_data_export_rule"
+description: |-
+  Create and manage Lacework Data Export Rules
+---
+
+# lacework\_data\_export\_rule
+
+Use this resource to export data collected from your Lacework account.
+For more information, see the [Data Export Rules documentation](https://docs.lacework.com/console/category/data-shares--export).
+
+## Example Usage
+
+#### Data Export Rule with Slack Data Export Channel
```

Comment:
wait what? I don't think you can use a Slack Alert Channel... did you test this?

Reply:
No, this is a copy pasta. The integration test uses S3